packages/extension-ui/src/components/Toast/Toast.tsx

```diff
@@ -55,7 +55,8 @@ export default styled(Toast)(
   padding: 16px;
   box-sizing: border-box;
   animation: toast 0.2s, toast  0.2s linear 1.4s reverse;
-  
+  z-index: 100;
```

Comment:
It might be beneficial to do a common file called 'zIndex'  or add it to theme
```js
 {
  toast: 100,
  splash: 110
}
```
etc. Then import in all places zIndex is needed. It's always nice to have this "zIndex" config for reference later and figuring out why something is over something else :)